examples/next-openai-pages/pages/chat/stream-chat-files/index.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable @next/next/no-img-element */
+import { useChat } from 'ai/react';
+import { useRef, useState } from 'react';
+
+export default function Page() {
+  const { messages, input, handleSubmit, handleInputChange, isLoading } =
+    useChat({
+      api: '/api/stream-chat',
+    });
+
+  const [attachments, setAttachments] = useState<FileList | undefined>(
```

Comment:
my take would be to call it "files", "setFiles" here (since we are on the file list level). Those files will then be used as attachments

Reply:
(then the uploaded `files` and the `attachments` on msgs are clearly distinguished)

Reply:
but we could also leave as is to have consistent examples